lsp-mode.el

```diff
@@ -7861,6 +7863,33 @@ archieve(e. g. when the archieve has multiple files)"
       (set-file-modes store-path #o0700)
       store-path)
      ((f-exists? store-path) store-path))))
+
+(defun lsp--find-latest-release-url (url &optional system-names)
```

Comment:
instead of system-name, can this be a regex instead? and this function will call into URL, extracting URL from the item that matches regex